tests/cucumber/features/channels/channel_creation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Channel creation
+  As an authenticated user
+  I want to create a channel
+
+  Background:
+    Given I am signed in
+    And I am on the new channels page
```

Comment:
I couldn't tell you. The "I am signed in" step is still navigating to the home page before it attempts anything, so I don't think that's related. It could just have been a side effect of cleaning the tests up? I'm not even sure what blame to look at...